recipes/express/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: express
+  version: "1.5.1"
+
+build:
+  number: 0
+
+source:
+  fn: express_1.5.1_linux_x64.tar.gz # [linux64]
+  url: https://depot.galaxyproject.org/software/express/express_1.5.1_linux_x64.tar.gz # [linux64]
+  fn: express_1.5.1_darwin_x64.tar.gz # [osx]
+  url: https://depot.galaxyproject.org/software/express/express_1.5.1_darwin_x64.tar.gz # [osx]
```

Comment:
bioconda compiles it's own binaries on on old image to make the binaries compatible with mainly every unix system out there (libc etc...). So it would be favourable to compile the package and not use pre-compiled once.